lib/index.d.ts

```diff
@@ -0,0 +1,104 @@
+// Type definitions for panel 0.10.0
```

Comment:
I totally agree with you, and I'm glad to see an outstanding TS issue on the matter. however, I think the documentation will really help with intellisense & maybe even cut down on the learning curve for panel for new developers. I'm inclined to keep as is until TS issue is resolved. thoughts?